ormar/queryset/actions/order_action.py

```diff
@@ -95,6 +111,40 @@ def _split_value_into_parts(self, order_str: str) -> None:
         self.field_name = parts[-1]
         self.related_parts = parts[:-1]
 
+    def _handle_field_nulls_mysql(self, field_name: str, result: str) -> str:
+        """
+        Generate the Final Query with handling mysql syntax for nulls value
+
+        :param field_name: string name of this field for order
+        :type field_name: str
+        :param result: query generated in previous stage without nulls value
+        :type result: str
+        :return: result of the final query by field name and direction and nulls value
+        :rtype: str
+        """
+
+        if not self.is_mysql_bool:
+            return result + f" nulls {self.nulls}"  # pragma: no cover
+
+        condition: str = "not" if self.nulls == "first" else ""  # pragma: no cover
+        return f"{field_name} is {condition} null, {result}"  # pragma: no cover
```

Comment:
If you split this condition into two the result now lacks prefix and will not work with nested models.

You would have in example `{prefix}{table_name}.{field_name} is null, {field_name} {self.direction}`.

The second field_name does not have the prefix.

Reply:
> If you split this condition into two the result now lacks prefix and will not work with nested models.
> 
> You would have in example `{prefix}{table_name}.{field_name} is null, {field_name} {self.direction}`.
> 
> The second field_name does not have the prefix.

Of course, I must mention that without this case, the test related to nested models will also pass correctly. Can you give an example of a possible error that may occur? Thanks

Reply:
When you have a parent model with two relations to the child model. So like items with `created_by` and `modified_by` that lead to the same `User` model. Each of those relations in joins has to have different prefixes as otherwise, you would have ambiguous column names.